docs/website/docs/plus/features/pipeline-runner.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "Dlt+ Runner"
+description: Run pipelines with the Dlt+ Runner
+keywords: ["runner", "pipeline", "retry", "trace"]
+---
+# Runner
+
+
+The dlt+ Runner provides a production-ready run command for your pipelines. It offers robust error handling, retry mechanisms, and 
+near atomic trace storage to destinations of your choice.
+The runner can be configured for each pipeline via your `dlt.yml`, or you can use it directly
+in your python code.
+
+## Key features
+
+- **Trace storage** to a configurable destination for debugging and monitoring pipeline executions
+- **Automatic retry policies** with configurable backoff strategies
+- **Clean state management** to ensure consistent pipeline runs
+
+## Configuration
+
+The Runner is configured through the `run_config` section in your `dlt.yml` file.
+
+### Complete configuration example
+
+```yaml
+pipelines:
+  my_pipeline:
+    source: my_source
+    destination: duckdb
+    run_config:
+      run_from_clean_folder: true
+      store_trace_info: true
+      retry_policy:
+        type: backoff
+        max_attempts: 5
+        multiplier: 2
+        min: 1
+        max: 10
+      retry_pipeline_steps: ["load"]
+```
+
+## Run from clean folder
+
+The `run_from_clean_folder` parameter controls whether the pipeline starts from a clean state before running.
+When enabled, the pipeline's working directory which holds the state, schema and any pending data are deleted 
+and state and schema are synchronized from the destination.
+
+```yaml
+pipelines:
+  my_pipeline:
+    run_config:
+      run_from_clean_folder: true
+```
+
+:::note
+The dlt+ runner behaves differently from `pipeline.run()` when there exists pending data in the pipeline's working directory:
+`pipeline.run()` will load only the pending data instead and needs to be invoked again for the given data.
+The dlt+ runner will also try finalizing the pending data, applying the retry policy and the trace settings, and then also
+try running with the given data.
+:::
+
+
+## Trace storage
+
+The `store_trace_info` parameter enables automatic storage of the pipelines trace, which can be used for for debugging, auditing, and monitoring pipeline runs.
```

Comment:
Should we describe what exactly is in the traces? Also, you can add a link to the Pydantic description of the traces